grunt/replace.js

```diff
@@ -346,8 +346,8 @@ module.exports = function(grunt) {
       options: {
         patterns: [
           {
-            match: /import\(/,
-            replacement: 'import_INTENTIONALLY_BROKEN('
+            match: /import\([^)]*\)/,
+            replacement: 'Promise.reject(new Error("INTENTIONALLY BROKEN: this build of intl-tel-input includes the utilities module inline, but it has incorrectly attempted to load the utilities separately. If you are seeing this message, something is broken!"))'
```

Comment:
The previous approach would cause some promises to never resolve, which shouldn’t be a problem for most users, but did make tests complicated. This rejection solves that and hopefully gives users (or future contributors) a little clearer description of what’s happening.